sky/skylet/providers/azure/azure-vm-template.json

```diff
@@ -88,6 +88,12 @@
             "metadata": {
                 "description": "OS disk size in GBs."
             }
+        },
+        "osDiskType": {
+            "type": "string",
+            "metadata": {
+                "description": "OS disk type."
```

Comment:
Is it able to make this choices of the three types, instead of the general string?